pvlib/irradiance.py

```diff
@@ -2915,3 +2917,68 @@ def dni(ghi, dhi, zenith, clearsky_dni=None, clearsky_tolerance=1.1,
             (zenith < zenith_threshold_for_zero_dni) &
             (dni > max_dni)] = max_dni
     return dni
+
+
+def component_sum_irradiance(solar_zenith,
+                             ghi=None,
+                             dhi=None,
+                             dni=None,
+                             dni_clear=None):
+    """
```

Comment:
```suggestion
    r"""
```
This will silence the stickler complaint in the latex stuff farther down